README.md

```diff
@@ -1,23 +1,96 @@
 # Libero walking skeleton
 
-This repo contains experiments that will comprise a walking skeleton of the Libero publishing system.
+This repo contains a walking skeleton of the Libero publishing system. It does not contain production-ready/polished code, but is a representation of how Libero can work.
 
-Execute `./example.sh` in your terminal to follow a demo.
+Execute `./example.sh` in your terminal to follow a demo. This shows:
 
-It contains:
+1. Creating an article
+2. Adding a new version
+3. Updating a version
+4. Deleting an article
 
 ## Applications
 
 These can be run using `docker-compose up -V`.
```

Comment:
Since there are many services here, one more level in the hierarchy (e.g. API-oriented vs user interface or public-oriented vs back office) may help understanding what they are

Reply:
It's in https://drive.google.com/drive/folders/1MUU8ewCUWN0lwq5i7NCbA9IGwEf2-oel if you want to tweak. Trying to find a balance between factual and complication!